### PR TITLE
Init Openshift Kapp Controller

### PR DESCRIPTION
**Pull request by theishshah**
Add initial yaml to enable prow for openshift operator framework kapp controller

### CODE REVIEW

**Comment by grokspawn** [comment]
Looks like you're missing the `make jobs` step from https://docs.ci.openshift.org/docs/how-tos/onboarding-a-new-component/#generating-prow-jobs-from-ci-operator-configuration-files
You will likely also have to run `make openshift-image-mirror-mappings` because you're adding a new image [here](https://github.com/openshift/release/pull/49737/files#diff-566f9e1b5a8bdc12eafd924c457b793b718aad51f07db047ece7f1a9d69efa02R25)

File: ci-operator/config/openshift/operator-framework-kapp-controller/openshift-operator-framework-kapp-controller-main.yml [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+    # Verify openshift dir
+    cd openshift && make verify-deps
+  container:
+    from: src
+- as: verify-manifests
+  commands: make -f openshift/Makefile verify-manifests
+  container:
+    from: src
+- as: verify-commits
+  commands: commitchecker --start ${PULL_BASE_SHA:-main}
```
Comment (m1kola) [inline ci-operator/config/openshift/operator-framework-kapp-controller/openshift-operator-framework-kapp-controller-main.yml:61]:
```suggestion
  commands: commitchecker --start ${PULL_BASE_SHA:-develop}
```

File: ci-operator/config/openshift/operator-framework-kapp-controller/openshift-operator-framework-kapp-controller-main.yml [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+    from: commitchecker
+- as: openshift-e2e-aws
+  skip_if_only_changed: ^docs/|\.md$|^(?:.*\/)?(?:\.gitignore|DOWNSTREAM_OWNERS|DOWNSTREAM_OWNERS_ALIASES|LICENSE|DCO|CODEOWNERS|PROJECT)|\.golangci\.yaml|\.goreleaser\.yml|\.markdownlint.yaml|^\.github|Tiltfile$
+  steps:
+    cluster_profile: aws
+    env:
+      FEATURE_SET: TechPreviewNoUpgrade
+    workflow: openshift-e2e-aws
+zz_generated_metadata:
+  branch: main
```
Comment (m1kola) [inline ci-operator/config/openshift/operator-framework-kapp-controller/openshift-operator-framework-kapp-controller-main.yml:72]:
```suggestion
  branch: develop
```

**Review by m1kola — COMMENTED** [review]
We use develop as a default branch in this repo. So need to keep an eye on places where we need to update these settings.

**Review by m1kola — CHANGES_REQUESTED** [review]
@theishshah we decided to change the direction and this is no longer needed. Let's close this PR please.

**Comment by grokspawn** [comment]
@theishshah close this since we are going a diff way?

**Pull request closed without merging**